### DISCUSSION TITLE
Access to translation

### DISCUSSION THREAD

**Discussion by maksimileean** [requarks/wiki#6133] [Help / Questions] [answered]
Hello, where and how can I get access to the translation web service?

I've tried https://gitter.im/Requarks/wiki , but 
![image](https://user-images.githubusercontent.com/114295776/217264677-f47b6f92-65d9-4716-bd5b-59b40dc18a08.png)

**Comment by NGPixel** [comment, marked as answer]
Have you [read the docs](https://docs.requarks.io/dev/translations)?
